README.md

```diff
@@ -296,9 +296,26 @@ You can use `config.getRemote()` and `config.setRemote()` for
 
 `npm test`
 
+## Best Practices
+
+Zero-config is designed to help you structure your config 
+files to support a number of production concerns. These best
+ practices reflect our approach and some of the reasons we 
+ designed Zero-config as we did.
+
+ - Most configuration should live in `common.json`
```

Comment:
we actually hate this pattern, most configuration should live in production.json :(